commands/new.js

```diff
@@ -0,0 +1,190 @@
+const path = require('path');
+const fs = require('fs-extra');
+const execa = require('execa');
+const inquirer = require('inquirer');
+
+const templatePath = path.resolve(__dirname, '../templates');
+
+const repoName = name => name.replace(/^@4c\//, '');
+
+async function setupGit(dest, { name }) {
+  await fs.copyFile(
+    path.join(templatePath, '.gitignore'),
+    path.join(dest, '.gitignore'),
+  );
+
+  await execa('git', ['init'], { cwd: dest });
+
+  await execa(
+    'git',
+    [
+      'remote',
+      'add',
+      'origin',
+      `git@github.com:4catalyzer/${repoName(name)}.git`,
```

Comment:
parametrize the org maybe?

Reply:
you think we'd use it for bfly or qsi? i already have qsi tooling, not sure i can write in terms of each other tho maybe